mlserver/settings.py

```diff
@@ -241,6 +241,12 @@ class Config:
     _custom_metrics_server_settings: Optional[dict] = None
     _custom_grpc_server_settings: Optional[dict] = None
 
+    cache_enabled: Optional[bool] = None
+    """Enable caching for the model predictions."""
+
+    cache_size: Optional[int] = None
+    """Cache size to be used if caching is enabled."""
```

Comment:
I was actually thinking the opposite here! In general, I find nullable types a bit of a pain sometimes (particularly when it comes to type checking - `mypy` complains a lot when you "forget" checking if something like `cache_size` is `None`). 

In this case, I think it would make sense to set them to a default value and make them non-nullable, i.e. something like

```python
  cache_enabled: bool = False
    """Enable caching for the model predictions."""

    cache_size: int = 100
    """Cache size to be used if caching is enabled."""
```

Reply:
Updated in [b30c41f](https://github.com/SeldonIO/MLServer/pull/1440/commits/b30c41f690076eb92d785e4268165ff7a36834aa)